src/harness/sas_interface.py

```diff
@@ -279,27 +279,14 @@ def PreloadRegistrationData(self, request):
     pass
 
   @abc.abstractmethod
-  def InjectExclusionZone(self, request):
+  def InjectExclusionZone(self, request): 
     """Inject exclusion zone information into SAS under test.
 
     Args:
-      request: A dictionary with key-value pairs that injects the information 
-      about single or multiple Exclusion Zones into the SAS under test. For 
-      more information about ZoneData please see the SAS-SAS TS (WINNF-16-S-0096).
-    """
-    pass
-
-
-  @abc.abstractmethod
-  def InjectZoneData(self, request):
-    """Inject PPA or NTIA zone information into SAS under test.
-
-    Args:
-      request: A dictionary with a single key-value pair where the key is
-        "record" and the value is ZoneData object to be injected into 
-        SAS under test. For more information about ZoneData please see 
-        the SAS-SAS TS (WINNF-16-S-0096).
-    """
+      request: A dictionary with the following key-value pairs:
+        "exclusionZone": A GeoJSON object defining the exclusion zone to be injected to SAS UUT.
```

Comment:
Please align with the testing API spec (the default config generation function is correct).

Reply:
Yes you   are right. Corrected it as per default config generation function